recipe/meta.yaml

```diff
@@ -1,64 +1,125 @@
 {% set name = "awscli" %}
-{% set version = "1.27.7" %}
-{% set hash = "faa8acf66eea21447b04f0d021c4539cd714b2770f1b80cd2222ec68e3313c94" %}
-
-{% set vmajor,vminor,vpatch = version.split('.') %}
-# ALWAYS DOUBLE-CHECK THESE OFFSETS AGAINST UPSTREAM setup.py
-{% set botocore_major = vmajor | int + 0 %}
-{% set botocore_minor = vminor | int + 2 %}
-{% set botocore_patch = vpatch | int + 0 %}
-{% set botocore_pin = botocore_major ~ '.' ~ botocore_minor ~ '.' ~ botocore_patch %}
+{% set version = "2.8.11" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
+# v2 is not on PyPI https://github.com/aws/aws-cli/issues/6785
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
-  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: {{ hash }}
+  url: https://github.com/aws/aws-cli/archive/{{ version }}.tar.gz
+  sha256: b37c64c6583153a420c7b80b34d62d33e317b40db561dfaeef027da083debb17
 
 build:
   number: 0
-  script: {{ PYTHON }} -m pip install . --no-deps -vv  # [not unix]
-  script: {{ PYTHON }} -m pip install . --no-deps -vv && mkdir -p $PREFIX/share/bash-completions/completions && echo complete -C aws_completer aws >$PREFIX/share/bash-completions/completions/aws  # [unix]
+  noarch: python
```

Comment:
does v2 dramatically change how the cli entrypoint is created on windows? if not, can't go noarch

Reply:
Sorry, you're right. I had a patch to add a `console_scripts` entrypoint in a previous dev version of the recipe. I've removed noarch.